apps/GatewayApi/Controllers/StatusController.cs

```diff
@@ -118,7 +119,7 @@ public GatewayModel.NetworkConfigurationResponse NetworkConfiguration()
                 transactionProcessorPackage: wellKnownAddresses.TransactionProcessorPackage,
                 metadataModulePackage: wellKnownAddresses.MetadataModulePackage,
                 royaltyModulePackage: wellKnownAddresses.RoyaltyModulePackage,
-                accessRulesPackage: wellKnownAddresses.AccessRulesPackage,
+                accessRulesPackage: wellKnownAddresses.RoleAssignmentModulePackage,
```

Comment:
hmmm shouldn't we rename that field ?:P I know it's breaking change but...

Reply:
Yes, we definitely should! This change is part of separate PR (with xrd_usd_price)